dapp/src/utils/web3.js

```diff
@@ -26,7 +25,6 @@ export function getEtherscanHost(web3React) {
   const chainIdToEtherscan = {
     1: 'https://etherscan.io',
     3: 'https://ropsten.etherscan.io',
```

Comment:
let's clean up ropsten since it is not used

Reply:
Removed in the latest commit